.pre-commit-config.yaml

```
@@ -25,7 +25,7 @@ repos:
   #                  footnote), values.yaml, .frigate templates for README.md
   #                  files.
   - repo: https://github.com/rapidsai/frigate/
-    rev: v0.6.0
+    rev: v0.7.0a
```

Let's bump to [v0.6.01 the git tag](https://github.com/rapidsai/frigate/tags), that i assume matches the [0.6.1 release on PyPI](https://pypi.org/project/frigate/#history).

```suggestion
    rev: v0.6.01
```